Makefile

```diff
@@ -57,14 +56,19 @@ package-goo:
 	goopack -output_dir ./dist <(envsubst < ./.build/googet/google-cloudops-opentelemetry-collector.goospec)
 	chmod -R 777 ./dist/
 
+# exporters
+.PHONY: build-exporters
+build-exporters:
+	bash ./.build/tar/build_exporters.sh
+
 # tarball
 
 .PHONY: build-tarball
-build-tarball: build package-tarball
+build-tarball: build build-exporters package-tarball
```

Comment:
Should `build-exporters` be optional? We need to provide a way to build a tarball without that don't we?

This is not necessarily required, but I think it would be even better if users could specify which exporters that want to include somehow (comma separated list?)

Another thought: shouldn't we clean `dist` before creating our tarball as there could be junk in there left over from other builds?

Reply:
Yes, the exporters should be configurable, but given the lack of time we agreed that letting the user choose which receivers they don't want simply by removing them in the config is a viable option for now! The exporters would just be useless/not used in those cases.

I will make it optional though and good point on cleaning the dist/ folder

Reply:
If users aren't going to use an exporter they wouldn't want to bundle it in the tar file at all. I guess its not a huge issue, but I would greatly prefer it if we at least had a make command for with exporters and without, which should be relatively trivial I think?

Reply:
Nvm just saw you already did that